docs/ogc/protocols/ogc-csw/README.md

```diff
@@ -17,11 +17,11 @@ tags:
 Overview - source
 The CSW service exposes the metadata records in XML format using the [OGC CSW protocol](https://docs.pycsw.org/en/latest/introduction.html#csw-support) 🌐
 
-[CSW Registry Service (Offline)](../../../static/pdfs/manuals/07-110r4_CSW-ebRIM_Registry_Service_-_Part_1_ebRIM_profile_of_CSW_v1.0.1.pdf "download") 📚
+[CSW Registry Service (Offline)](../../../../static/pdfs/manuals/07-110r4_CSW-ebRIM_Registry_Service_-_Part_1_ebRIM_profile_of_CSW_v1.0.1.pdf "download") 📚
```

Comment:
![image](https://github.com/MapColonies/developer-portal/assets/18442671/224d797f-42e4-4256-a7e4-b0b534aa81a4)

Do we want to link our available  CSW profiles here?

Reply:
Added.